docs/coordinates/performance.inc.rst

```diff
@@ -56,3 +56,157 @@ properties::
 
 ..
   EXAMPLE END
+
+
+Improving Performance for Arrays of ``obstime``
+-----------------------------------------------
+
+The most expensive operations when transforming between observer-dependent coordinate
+frames (e.g. ``AltAz``) and sky-fixed frames (e.g. ``ICRS``) are the calculation
+of the orientation and position of Earth.
+
+If |skycoord| instances with a large ``obstime`` array are transformed,
+these calculations can be sped up by factors up to 100, whilst still keeping micro-arcsecond precision,
+by utilizing interpolation instead of calculating Earth orientation parameters for each individual point.
+
+This can be achieved through the ``erfa_astrom`` state and the ``ErfaAstromInterpolator``
```

Comment:
I'd just delete this sentence (if not, do link the text properly with either the above or the below, and ensure that `erfa_astrom` and `ErfaAstromInterpolator` are links, as in `~astropy.coordinates.erfa_astrom.erfa_astrom`)

Reply:
done